.github/workflows/blackduck-scan.yml

```diff
@@ -0,0 +1,45 @@
+name: BlackDuck-Scan-Workflow
+
+on:
+  # Triggers the workflow on push request events but only for the master branch
+  pull_request:
+    branches: [main]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  scan:
+    name: BlackDuck-scan
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name == 'EnterpriseDB/pg-backup-api'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2.1.0
+        with:
+          java-version: 8
+          distribution: "zulu"
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install project
+        working-directory: ./pg_backup_api
+        run: |
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install -r requirements.txt
+          python -m pip install -e .
+      # -- Build with steps and requirements
+      - name: Synopsys Detect
+        uses: blackducksoftware/github-action@2.0.0
+        with:
+          args: '--blackduck.url="${{ secrets.BLACKDUCK_URL}}" --blackduck.api.token="${{ secrets.BLACKDUCK_API_TOKEN}}" --detect.project.name="pg-backup-api" --detect.project.version.name="CI/CD" --detect.risk.report.pdf=true --detect.excluded.detector.types="npm,cpan,nuget,git,maven,lerna" --detect.blackduck.signature.scanner.license.search --detect.source.path="pg_backup_api"'
+
+      # Just print the end of scan
+      - name: End of Scan
+        run: |
+          echo Finished scanning.
```

Comment:
I can't figure out how to put a comment on an empty file, but I assume pg_backup_api/foo was for testing and will be deleted before merge?

Reply:
Good catch - this was supposed to be deleted but I forgot about it when I squashed the commits.